src/desktop/apps/auction/components/layout/__tests__/ConfirmRegistrationModal.jest.tsx

```diff
@@ -0,0 +1,95 @@
+import renderTestComponent from "desktop/apps/auction/__tests__/utils/renderTestComponent"
+import { ConfirmRegistrationModal } from "../ConfirmRegistrationModal"
+import { act } from "react-dom/test-utils"
+describe("Confirm Registration Modal", () => {
+  beforeAll(() => {
+    jest.spyOn(history, "replaceState")
+  })
+
+  afterAll(() => {
+    jest.restoreAllMocks()
+  })
+
+  describe("sanitizing the URL", () => {
+    xit("removes /confirm-registration from the url", () => {
+      document.title = "big time"
+      window.location.pathname = "/auction/auction-one/confirm-registration"
+
+      act(() =>
+        renderTestComponent({
+          Component: ConfirmRegistrationModal,
+          options: { renderMode: "mount" },
+        })
+      )
+
+      expect(history.replaceState).toHaveBeenCalledWith(
+        {},
+        "big time",
+        "/auction/auction-one"
```

Comment:
Could not get this 3rd parameter to come through so I disabled this test - always `/`. It seems that just setting `location.pathname` in the test code does not work. I feel like there is some trivial bit that has slipped my mind.